marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/runEmptyMapping.sjs

```diff
@@ -0,0 +1,37 @@
+const mapping = require("/data-hub/5/builtins/steps/mapping/entity-services/main.sjs");
+const test = require("/test/test-helper.xqy");
+
+let assertions = [];
+
+//Only 'id' is mapped. Other properties shouldn't be present in entity instance
+function runStructuredPropertyMapping () {
+  let doc = fn.head(xdmp.unquote(`{"id":1,"customer":{"firstName":"Oralia"},"items":[{"name":"Voltsillam"},{"name":"Latlux"}]}`));
+  let result = mapping.main({uri: '/test.json', value: doc}, {
+    mapping: {name: 'OrdersMapping', version: 4},
+    outputFormat: 'json'
+  }).value.root;
+  let instance = result.envelope.instance;
+  assertions = assertions.concat([
+    test.assertEqual(null, instance.OrderType.customer),
+    test.assertEqual(null, instance.OrderType.items),
+    test.assertEqual("1", fn.string(instance.OrderType.id))
+  ]);
+}
+
+//None of the entity properties are mapped, so 'CustomerType' shouldn't be present in entity instance
```

Comment:
This is fine - no need to change - but a comment like this is best expressed as the assertion message - e.g. 

    test.assertEqual(null, instance.CustomerType, "Since none of the Customer properties are mapped, it should not exist in the entity instance");

There are two benefits to this:

1. If the test fails, we get a nice message as opposed to just that something isn't null
1. If the logic changes, or if the code is reorganized, the assertion message is almost guaranteed to stay in sync with the code, while the comment may not get updated or may not get moved